### ISSUE TITLE
fix: template variable was not injected....newest unplugin-vue2-script-setup required

### ISSUE THREAD

**Issue by zcqno1** [nuxt-community/composition-api#609]
**🐛 The bug**
What isn't working? Describe what the bug is.

template with wrapped `{{ }}` was not able to be recognized by older version of unplugin-vue2-script-setup.
```vue
<template>
  <div>
    <span class="test test test test test ml-3 text-light text-xs">{{
      dialogTitle
    }}</span>
  </div>
</template>
```

![image](https://user-images.githubusercontent.com/11766057/155126138-430c4f52-cf17-4bb7-b03a-6eadf66b8097.png)

variables between wrapped `{{ }}` are not injected to the vm.

After upgrading unplugin-vue2-script-setup to the newest version manually, the problem is solved.

BTW, the version of unplugin-vue2-script-setup in the latest @nuxtjs/composition-api is `0.7.3` while its own newest version is `0.9.3`.

**🌈 Expected behaviour**
Template parse should work as usual.
A new release with the newest unplugin-vue2-script-setup dependency should be arranged.

**🪜 Workaround**

adjust the template to avoid wrapped `{{ }}`, like:

```vue
<template>
  <div>
    <span class="test test test test test ml-3 text-light text-xs">
      {{ dialogTitle }}
    </span>
  </div>
</template>
```

**Comment by danielroe** [comment]
I've released @nuxtjs/composition-api@0.32.0 - let me know if this solves the issue for you.

**Comment by zcqno1** [comment]
New release solves this issue !!! :

Amazing，it's really so quick to get your response.⚡
Thanks for your job. 👍  @danielroe 

btw, a reminder, `yarn dev` should be run to restart the nuxt app if your app was kept in hmr mode or  just hold on to upgrade the @nuxtjs/composition-api.
